00-programming-fundamentals/git-intro-to-git.md

```diff
@@ -234,6 +234,16 @@ With these two lines Git is showing us first what file we have modified in our p
 
 If we were to run `git log` again, we would see our new commit at the top of the list.
 
+## Summary
+
+command                 | description
+------------------------|-----------------------------------------------------
+`git clone $GITHUB_URI` | Make a local copy of `$GITHUB_URI` from github.
```

Comment:
Why are you calling this a URI vs URL? Do you think the distinction is going to be obvious to the students (Devin)?

Reply:
In the section on cloning it was called a URI.